File: 5° semestre/Programação para dispositivos moveis II/aula-25-09-26/atividades pré prova/5 - PizzariaPagamento/projeto5/app/src/main/java/br/gov/sp/cps/pizzariapagamento/MainActivity.java
package br.gov.sp.cps.pizzariapagamento;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.widget.CheckBox;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    private CheckBox cb1, cb2, cb3, cb4, cb5;
    private Button btn1;
    ArrayList<String> sabores = new ArrayList<>();
    private double[] precoSabores = {36.90, 38.90, 34.90, 42.90, 41.90};
    private double parcialPreco;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        cb1 = findViewById(R.id.cb1);
        cb2 = findViewById(R.id.cb2);
        cb3 = findViewById(R.id.cb3);
        cb4 = findViewById(R.id.cb4);
        cb5 = findViewById(R.id.cb5);
        btn1 = findViewById(R.id.btn1);

        btn1.setOnClickListener(v -> {
            sabores.clear();
            parcialPreco = 0;
            if (cb1.isChecked()) {
                sabores.add(cb1.getText().toString());
                parcialPreco += precoSabores[0];
            }
            if (cb2.isChecked()) {
                sabores.add(cb2.getText().toString());
                parcialPreco += precoSabores[1];
            }
            if (cb3.isChecked()) {
                sabores.add(cb3.getText().toString());
                parcialPreco += precoSabores[2];
            }
            if (cb4.isChecked()) {
                sabores.add(cb4.getText().toString());
                parcialPreco += precoSabores[3];
            }
            if (cb5.isChecked()) {
                sabores.add(cb5.getText().toString());
                parcialPreco += precoSabores[4];
            }

            Intent intent = new Intent(this, MainActivity2.class);
            intent.putStringArrayListExtra("sabores", sabores);
            intent.putExtra("precoParcial", parcialPreco);
            startActivity(intent);
        });

    }
}

File: 3° semestre/Programação para dispositivos Moveis I/aula-24-10-08/ListaCompras/MainActivity.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val listViewProduto = findViewById<ListView>(R.id.listViewProdutos)
        val btnInserir = findViewById<Button>(R.id.btnInserir)
        val txtProduto = findViewById<EditText>(R.id.txtProduto)
        val produtosAdapter = ArrayAdapter<String>(this, android.R.layout.simple_list_item_1)

        listViewProduto.adapter = produtosAdapter

        btnInserir.setOnClickListener {
            val produto = txtProduto.text.toString()
            if (produto.isNotEmpty()) {
                produtosAdapter.add(produto)
                txtProduto.text.clear()
            } else {
                txtProduto.error = "Coloque um Produto"
            }
        }

        listViewProduto.setOnItemClickListener { adapterView: AdapterView<*>, view, position: Int, id ->
            val item = produtosAdapter.getItem(position)
            produtosAdapter.remove(item)
        }
    }
}
